extensions/amp-position-observer/validator-amp-position-observer.protoascii

```diff
@@ -39,6 +39,7 @@ tags: {  # <amp-position-observer>
     value_regex: "^([0]*?\\.\\d*$|1$|0$)|([0]*?\\.\\d*|1|0)\\s{1}([0]*?\\.\\d*$|1$|0$)"
   }
   attrs: { name: "target" }
+  attrs: { name: "once" }
```

Comment:
name: "once", value: ""

Reply:
perhaps it should be called `runonce`, `run-once` or `run_once`? Or is it obvious to users that `once` means it runs once?

Reply:
True `once` isn't clear enough. 

How about `norepeat` to mimic `<amp-bodymovin-animation>` and `<amp-gfycat>`?

Reply:
Changed the name to `norepeat`.

Reply:
I liked `once` TBH. Matches the new  `addEventListener` spec in JS.

Reply:
Oohh if there is a parallel in the wild then I will revert to `once`.